src/protagonist/API.Tests/Integration/CustomerQueueTests.cs

```diff
@@ -558,6 +558,30 @@ public async Task Post_CreateBatch_201_IfLegacyModeEnabled()
         response.StatusCode.Should().Be(HttpStatusCode.Created);
     }
     
+    [Fact]
+    public async Task Post_CreateBatch_201_IfLegacyModeEnabled_MembersEmpty()

```

Comment:
Would `Post_CreateBatch_201_IfLegacyModeEnabled_AndMembersEmpty()` be slightly clearer as a test name?